arkouda/numeric.py

```diff
@@ -2248,3 +2255,305 @@ def putmask(pda: pdarray, mask: Union[bool, pdarray], values: pdarray):
         result = result[:-(reduction)]
 
     pda[:] = where(mask, result, pda)  # pda[:] = allows us to return modified value
+
+
+# after this is working, add comments and docstring
```

Comment:
Oops.  Line 2260 is one of those infamous "comments I forgot to remove."

Reply:
Removed it.